src/components/Column/index.tsx

```diff
@@ -5,7 +5,7 @@ import { Theme } from "types/global";
 const Column: React.FC = props => (
   <div
     css={({ ...theme }: Theme) => ({
-      flexBasis: "calc((42rem - 100%) * 9999)",
+      flexBasis: "calc((30rem - 100%) * 9999)",
```

Comment:
should this value be put into the theme file?

Reply:
Possibly but it is only used in this one spot so not sure if it is necessary